sympy/ntheory/primetest.py

```diff
@@ -547,9 +547,11 @@ def isprime(n):
         return True
     if n < 2 or (n % 2) == 0 or (n % 3) == 0 or (n % 5) == 0:
         return False
+    if n % 6 != 1 and n % 6 != 5:
```

Comment:
Anything that makes it through the `(n % 2) == 0 or (n % 3) == 0 or (n % 5) == 0` test has this property:
```python
>>> set([n%6 for n in range(1, 10**4) if not(n < 2 or (n % 2) == 0 or (n % 3) == 0 or (n % 5) == 0)])
{1, 5}
```

Reply:
Thank you for the clarification , could you look at this pr  #16212  please